src/markdown-pages/docs/managers/mq-manager.md

```diff
@@ -0,0 +1,188 @@
+---
+path: "/docs/managers/mq-manager"
+title: "MQ Manager"
+---
+
+**Alpha**
+
+## Overview
+This Manager provides the ability to connect a test to an existing IBM MQ queue manager, enabling one or more messages to be written to and read from an existing IBM MQ queue. <br><br> 
+
+
+## Annotations
+
+The following annotations are available with the MQ Manager
+<details>
+<summary>MQ Manager</summary>
+
+| Annotation: |MQ Manager |
+| --------------------------------------- | :------------------------------------- |
+| Name: | @QueueManager |
+| Description: | The <code>@QueueManager</code> annotation represents the name of the IBM MQ queue manager |
+| Attribute: `name` |  The name of the IBM MQ queue manager |
+| Attribute: `host` |  The host name of the system on which the IBM MQ queue manager is running |
+| Attribute: `port` |  The port number to connect to the IBM MQ queue manager |
+| Attribute: `channel` |  The name of the channel used to move messages  |
+| Attribute: `manager` |  The Galasa implementation of the IBM MQ queue manager |
```

Comment:
these are not attributes of the annotation - but CPS properties

Reply:
added annotation attributes